Resources/Prototypes/Body/Organs/arachnid.yml

```
@@ -164,6 +164,8 @@
   description: "Filters toxins from the bloodstream."
   categories: [ HideSpawnMenu ]
   components:
+  - type: Organ # Shitmed
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)